execute/executetest/transformation.go

```diff
@@ -109,7 +110,13 @@ func ProcessTestHelper(
 	sort.Sort(SortedTables(got))
 	sort.Sort(SortedTables(want))
 
-	if !cmp.Equal(want, got, floatOptions) {
+	opts := make([]cmp.Option, 0, len(floatOptions)+len(options))
+	if len(options) > 0 {
+		opts = append(opts, options...)
+	} else {
+		opts = append(opts, floatOptions...)
+	}
+	if !cmp.Equal(want, got, opts...) {
```

Comment:
So this will replace the old options there were there... Can you call this out in a comment on `ProcessTestHelper`? Maybe the parameter should be named `alternateFloatOptions` to clarify that it will take precedence over the default behavior.

Reply:
what if we rename [floatOptions](https://github.com/influxdata/flux/blob/92b442939403cb3e159a42bc17f4e668d735af32/execute/executetest/transformation.go#L34) to defaultFloatOptions, that way it would be clear if nothing passed in the option, defaultFloatOptions will be used.

Reply:
Sure, that works. Seems like the parameter should be named `floatOptions` in that case. Since that's all it's really used for?

Reply:
Yes, I will change the param name. Thanks